lib/iris/tests/test_basic_maths.py

```diff
@@ -39,11 +39,8 @@
 class TestBasicMaths(tests.IrisTest):
     def setUp(self):
         self.cube = iris.tests.stock.global_pp()
-        # We require to preserve the cube fill_value
-        # across the cube data setter operation.
-        fill_value = self.cube.fill_value
-        self.cube.data = self.cube.data - 260
-        self.cube.fill_value = fill_value
+        self.cube.replace(self.cube.data - 260,
+                          fill_value=self.cube.fill_value)
```

Comment:
I've started another project so that we can capture in the backlog all those important points that we want to address in user and developer documentation